calculator.html

```diff
@@ -4,9 +4,21 @@
 
 
  <h1> Calculator </h1>
- <textarea id ="input1"></textarea>   
- <textarea id ="input2"></textarea>
- <h2 id ="input3">  </h2>
+ <textarea id =input1></textarea>   
+ <textarea id =input2></textarea>
+ <p id =answer> </p>
 <p>Click the calculate button to get your result!</p>
-<button type="button" onclick="myFunction()">Calculate!</button>
-<a href="/anthony-s-blog/">Testing back link</a></body>
\ No newline at end of file
+<button type="button" onclick="addition()">Calculate!</button>
+
+<script>
+function addition() {   
+    var x = parseInt(document.getElementById("input1").value);
+    var y = parseInt(document.getElementById("input2").value);
```

Comment:
Can you change your variable names here to be descriptive? Also, please change these vars to lets.